### PR TITLE
Blog post for announcing Oct workshop

### PR DESCRIPTION
**Pull request by jennaswa**

### CODE REVIEW

File: _posts/2023-07-31-workshops.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+layout: post
+title: Online Training Workshop Series
+---
+
+MDAnalysis is excited to be kicking off a series of free online training workshops in Fall 2023. The first will be held on **October 25, 2023 from 15:00-18:30 UTC**. The next workshop will be organized for Australasian time zones and will be announced shortly.
+
+This remote workshop will consist of an introduction to the MDAnalysis Python library, as well as an opportunity for hands-on learning through basic MDAnalysis tutorials. The workshop will be delivered to a small group to allow interactive discussions, questions, and participant engagement. It is designed for beginners to MDAnalysis who already have a working knowledge of Python and working with shell and notebook environments.
+
+# Register soon (places limited)
+If you are interested in participating in this workshop, fill out [this short application][eventbrite] form before **August 30, 2023**. Selected participants will be notified by September 27.
```
Comment (orbeckst) [inline _posts/2023-07-31-workshops.md:11]: as discussed: move deadline to middle of September and notification to early October

File: _posts/2023-07-31-workshops.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+MDAnalysis is excited to be kicking off a series of free online training workshops in Fall 2023. The first will be held on **October 25, 2023 from 15:00-18:30 UTC**. The next workshop will be organized for Australasian time zones and will be announced shortly.
+
+This remote workshop will consist of an introduction to the MDAnalysis Python library, as well as an opportunity for hands-on learning through basic MDAnalysis tutorials. The workshop will be delivered to a small group to allow interactive discussions, questions, and participant engagement. It is designed for beginners to MDAnalysis who already have a working knowledge of Python and working with shell and notebook environments.
+
+# Register soon (places limited)
+If you are interested in participating in this workshop, fill out [this short application][eventbrite] form before **August 30, 2023**. Selected participants will be notified by September 27. 
+
+*We have a small number of bursaries to enable participation for researchers from underrepresented groups who are facing financial barriers*; these can be applied for when completing the application form.
```
Comment (orbeckst) [inline _posts/2023-07-31-workshops.md:13]: Do we need bursaries for online workshops? Is this for being able to take time off or paying for child care? (Sorry, I assume that was discussed so please feel free to ignore my comment.)

**Review by orbeckst — CHANGES_REQUESTED** [review]
Change deadline, otherwise good.

**Review by ianmkenney — DISMISSED** [review]
Pending @orbeckst's notes, everything looks good from here

File: _posts/2023-07-31-workshops.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Online Training Workshop Series
+---
+
+MDAnalysis is excited to be kicking off a series of free online training workshops in Fall 2023. The first will be held on **October 25, 2023 from 15:00-18:30 UTC**. The next workshop will be organized for Australasian time zones and will be announced shortly.
```
Comment (IAlibay) [inline _posts/2023-07-31-workshops.md:6]: Are we 100% on the austalasian edition or should we be saying that we are looking into it?

**Review by IAlibay — APPROVED** [review]
Just the one question, otherwise lgtm (pending Oliver's changes)

File: _posts/2023-07-31-workshops.md (hunk 3 above)
Comment (jennaswa) [reply _posts/2023-07-31-workshops.md]: Softened the language since we don't actually have a date on the calendar for this yet

**Comment by jennaswa** [comment]
@richardjgowers @micaela-matta - Before we send this off, want to just confirm that the October 25th date works for you both?

**Comment by richardjgowers** [comment]
@jennaswa yep from me

File: _posts/2023-07-31-workshops.md (hunk 2 above)
Comment (orbeckst) [reply _posts/2023-07-31-workshops.md:13]: 🤣  "Genau" ... you've been living in 🇩🇪 ...

**Review by orbeckst — APPROVED** [review]

**Comment by IAlibay** [comment]
Please feel free to merge when you want @jennaswa !

**Review by micaela-matta — APPROVED** [review]

**Pull request merged**